exemples/nodeAPI/app.ts

```diff
@@ -0,0 +1,44 @@
+import * as express from 'express';
+    // Creates and configures an ExpressJS web server.
+    class App {
+      // ref to Express instance
+      public express: express.Application;
+      //Run configuration methods on the Express instance.
+      constructor() {
+        this.express = express();
+        this.middleware();
+        this.routes();
+      }
+      // Configure Express middleware.
+      private middleware(): void {
+      }
+
+      // Configure API endpoints.
+      private routes(): void {
+        /* This is just to get up and running, and to make sure what we've got is
```

Comment:
Now that you understand commits and Pull Requests, this is the type of comment that no longer needs to be in the source. The problem here is it might stay for longer than needed and gives wrong information later in the project. It's also not necessarily evident to the people using the server later.